feature/selector.py

```diff
@@ -511,6 +515,78 @@ def benchmark(selectors: Dict[str, Union[SelectionMethod.Correlation,
     check_true(selectors is not None, ValueError("Benchmark selectors cannot be none."))
     check_true(data is not None, ValueError("Benchmark data cannot be none."))
 
+    if cv is None:
+        return _bench(selectors=selectors,
+                      data=data,
+                      labels=labels,
+                      output_filename=output_filename,
+                      drop_zero_variance_features=drop_zero_variance_features,
+                      verbose=verbose)
+    else:
+
+        # Create K-Fold object
+        kf = KFold(n_splits=cv, shuffle=True, random_state=Constants.default_seed)
```

Comment:
should not seed be a parameter of the caller function?

Reply:
Added in this [commit](https://github.com/fidelity/selective/pull/3/commits/f92f0de5d005ba7359d39d1f1e3be101237c03fa).